frontend/src/components/ShootWorkers/GWorkerConfiguration.vue

```diff
@@ -56,7 +56,13 @@ SPDX-License-Identifier: Apache-2.0
               :completion-paths="['spec.properties.provider.properties.workers', 'spec.properties.provider.properties.infrastructureConfig']"
               hide-toolbar
               animate-on-appear
-            />
+              alert-banner-identifier="workerEditorWarning"
+            >
+              <template #modificationWarning>
+                By modifying the resource directly you may cause serious problems in your cluster.
+                We cannot guarantee that you can solve problems that result from using worker resource editor incorrectly.
```

Comment:
or, instead of saying "you cannot solve problems" say that "some configuration might be irreversible" and could cause serious problems..

Reply:
Yes I basically copied the text from `GShootItemEditor`. Let's discuss if we can find a better phrase

Reply:
Proposal:
Directly modifying this resource can result in irreversible configurations that may severely compromise your cluster's stability and functionality.
Use resource editor with caution.